nkms/keystore/keypairs.py

```diff
@@ -200,3 +200,11 @@ def serialize_privkey(self) -> bytes:
                           constants.PRIV_KEY_BYTE +
                           self.privkey)
         return serialized_key
+
+
+class PublicKey(bytes):
+    _EXPECTED_LENGTH = 66
+    _METABYTES_LENGTH = 2
+
+    def without_metabytes(self):
+        return self[self._METABYTES_LENGTH::]
```

Comment:
Doesn't matter, but I think `[self._METABYTES_LENGTH:]` will be sufficient

Reply:
Somehow, I got on a kick of using two, thinking it's more explicit, but perhaps it's just merely less terse.

Reply:
Interesting. I'm totally fine with it though